README.md

```diff
@@ -43,7 +43,14 @@ Trigger when a new issue appears top of the search results of a JQL query (like
 
 ![CheckIssue](images/CheckIssue.png)
 
-
++ Check Issue (statuses) - Polls Jira to check the status of a Issue. It will complete once the issue is in the correct state
```

Comment:
1. "a Issue" -> "an Issue"
2. We mix use of "state" and "status".  I would recommend using one consistently.